### ISSUE TITLE
negative powers

### ISSUE THREAD

**Issue by RobinHankin** [RobinHankin/mvp#7]
it is very difficult to create a ```mvp``` object with negative powers; the mpoly package does not implement negative powers so idiom such as ```as.mvp("1+x^-5")``` does not work as desired.

I am not sure how to proceed here.

**Comment by RobinHankin** [comment]
following works, but is clunky.  Not 100% sure what I want to achieve here.  What kind of idiom would be desirable here?

```
> p <- as.mvp("1+jj^5")
> p %<>% subsmvp("jj",mvp("x",-1,1))
> p
mvp object algebraically equal to
1  +  x^-5
> 
```

**Comment by RobinHankin** [comment]
somewhat fixed in 1da4423
